### PR TITLE
core: use dedicated loader for es6 modules

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue: 
All previous attempts didn't worked.
When we're outside the monorepo, the way modules are resolved cause modules not being transpiled.

## What I did
Decicated babel loader

## How to test
IE 11 OUTSIDE the monorepo

### CODE REVIEW

**Comment by tooppaaa** [comment]
https://github.com/obahareth/are-you-es5/issues/31

File: lib/core/src/server/common/babel.js [hunk 1]
```diff
@@ -11,6 +11,26 @@ function createProdPresets() {
   ];
 }
 
+export const plugins = [
+  require.resolve('@babel/plugin-transform-shorthand-properties'),
+  require.resolve('@babel/plugin-transform-block-scoping'),
+
+  // [require.resolve('@babel/plugin-proposal-class-properties'), { loose: true }],
```
Comment (ndelangen) [inline lib/core/src/server/common/babel.js:18]: why is this commented out?

**Comment by tooppaaa** [comment]
@ndelangen Hope this is the last time I ever have to open IE :)

**Review by ndelangen — APPROVED** [review]
code looks good

File: lib/core/src/server/config/utils.js [hunk 2]
```diff
@@ -26,15 +26,7 @@ const projectRoot = () => {
...
-// Run `npx are-you-es5 check lib/core -r`
-// to get this regexp
-export const excludePaths = [
-  /[\\/]node_modules[\\/](?!(@storybook\/node-logger|are-you-es5|better-opn|boxen|chalk|commander|find-cache-dir|find-up|fs-extra|json5|node-fetch|pkg-dir|resolve-from|semver)[\\/])/,
-  // Never transpile again the following:
-  /[\\/]node_modules[\\/]webpack/,
-  /[\\/]node_modules[\\/]core-js/,
-  /[\\/]node_modules[\\/]regenerator-runtime/,
-];
+export const excludePaths = [/node_modules/];
```
Comment (ndelangen) [inline lib/core/src/server/config/utils.js:29]: Should use this everywhere OR nowhere IMHO

Comment (ndelangen) [reply lib/core/src/server/config/utils.js:29]: leaning towards, nowhere.

**Pull request merged**